changelogs/changelog.yaml

```diff
@@ -489,6 +489,7 @@ releases:
     changes:
       major_changes:
       - All modules now support SSL over HTTPS and socket level timeout.
+      - HTTPS SSL certificate validation is a ``breaking change`` and needs an update to the playbooks.
```

Comment:
Please see my previous comment for [CHANGELOG.rst](https://github.com/dell/dellemc-openmanage-ansible-modules/pull/359/files#r799453816)